docs/ocis/identity-provider/oidc/flow.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Flow
+weight: 40
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/identity-provider/oidc
+geekdocFilePath: flow.md
+---
+
+In Infinite Scale, authentication can follow one of the three methods described on the [official site](https://openid.net/specs/openid-connect-core-1_0.html#rfc.section.3):
+1. Authorization Code Flow
+2. Implicit Flow
+3. Hybrid Flow
+
+To authenticate using OIDC, both `client_id` and `client_secret` are essential. For oidc request, desktop-client `client_id` and `client_secret` can be used.
```

Comment:
> For oidc request,

The statement is not clear in regards to the context. Tests, setup, etc?
Only the desktop client is named but we have more like ios and android clients.

Yes, we refer below to the server docs, but I *highly* recommend, if that would not explode the page size, writing stuff down here. I can help if I have more context about what is intended to transport?